bitnami/wildfly/templates/deployment.yaml

```diff
@@ -84,19 +84,9 @@ spec:
             - name: mgmt
               containerPort: 9990
           livenessProbe:
-            httpGet:
-              path: /
-              port: http
-            initialDelaySeconds: 120
-            timeoutSeconds: 5
-            failureThreshold: 6
+
           readinessProbe:
-            httpGet:
-              path: /
-              port: http
-            initialDelaySeconds: 30
-            timeoutSeconds: 3
-            periodSeconds: 5
+            {{- include "wildfly.tplValue" ( dict "value" .Values.readinessProbe "context" $) | nindent 12 }}
```

Comment:
I deliberately did not use this pattern as it would not allow for the full range of possible lifeness/readiness probes.  As one example it would not allow for TCP probes https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-tcp-liveness-probe

Reply:
From my point, the thing here is, is actually useful to configure the liveness probe's type in this case? This will only check if the application is properly running, and that will be performed just checking the root endpoint throw HTTP. That endpoint should always exist since it is the root one (correct me if I am wrong), so I think the httpGet using `/` should be always a valid `livenessProbe`.
WDYT?

Reply:
This is true but in a real life use case we don't really care if Wildfly is up if the application running on the Wildfly pod has crashed right? The liveness probe should really tell us the application running on wildfly is healthy. That said I wouldn't be too surprised if wildfly has a default healthcheck, perhaps [here](https://wildfly-swarm.gitbooks.io/wildfly-swarm-users-guide/content/advanced/monitoring.html)(?) but that might require some application refactoring as it looks swarm specific.

Reply:
>The liveness probe should really tell us the application running on wildfly is healthy.

I am not a Wildfly expert, I supposed the application will be exposed at `/`, isn't that correct?

Reply:
No, in the case of an application failing to load I'm fairly certain Wildfly has a default page it shows welcoming you that throws a wrench in that. Also since Wildfly apps can use any number of protocols assuming httpGet is probably not valid either.

A general solution might be able to leverage an exec based healthcheck and the marker files https://docs.wildfly.org/20/Admin_Guide.html#marker-files. But that would only tell you if wildfly had failed to deploy something rather than, say a code failure. Might be worth looking into as a readiness or startup probe but I suppose that's a different matter.

Reply:
Thank you for the clarifications @kav ,
In that case, let's leave this customizable as @grinnellian did.